odl/contrib/fom/supervised.py

```diff
@@ -90,9 +102,16 @@ def mean_absolute_error(data, ground_truth, mask=None, normalized=False):
     ground_truth : `Tensor`
         Reference to compare ``data`` to.
     mask : `array-like`, optional
-        If given, ``data * mask`` is compared to ``ground_truth * mask``.
+        If given, ``data * mask`` is compared to ``ground_truth * mask``,
```

Comment:
This is importantly different from `data[mask]` vs `ground_truth[mask]` (this is biased towards 0), perhaps it should be mentioned